src/src/WebServer/DevicesPage.cpp

```diff
@@ -1262,23 +1262,22 @@ void devicePage_show_task_values(taskIndex_t taskIndex, deviceIndex_t DeviceInde
 
   if (!Device[DeviceIndex].Custom && (valueCount > 0))
   {
-    addFormSubHeader(F("Values"));
     html_end_table();
     html_table_class_normal();
-
     // table header
-    addHtml(F("<TR><TH style='width:30px;' align='center'>#"));
-    html_table_header(F("Name"));
+    addFormHeader(F("Values"));
+
+    addHtml(F("<TR><TD><H3>#</H3></TD><TD><H3>Name</H3></TD>"));
 
     if (Device[DeviceIndex].FormulaOption)
     {
-      html_table_header(F("Formula"), F("EasyFormula"), 0);
+       addHtml(F("<TD><H3>Formula</H3></TD>"));
     }
 
     if (Device[DeviceIndex].configurableDecimals())
     {
-      html_table_header(F("Decimals"), 30);
-    }
+       addHtml(F("<TD style='width:30px;''><H3>Decimals</H3></TD>"));
```

Comment:
The general idea of the separate functions like `html_table_header` is to reduce duplicate HTML strings in the code as it can add up really fast and thus create larger binary files.
So please try to fix it then in those functions, or if you really need to do things differently, then add another function for it.
But I guess if it is wrong here, it might be wrong on a lot of other places.

Also I doubt the double single quote is correct HTML.

Reply:
I understand... but i thought (apart from the fact that at this point i am not able to create a function ) since the valuetable is the only place where it appears its not so bad.

"But I guess if it is wrong here, it might be wrong on a lot of other places."
what do you mean by that?

Reply:
> "But I guess if it is wrong here, it might be wrong on a lot of other places."
> what do you mean by that?

Well since this function is called from a lot of places, I guess other tables may be wrong also.

The function does generate code like this:
```html
<TH style='width:30px;'>Decimals</TH>
```
But you now change it to something like this:
```html
<TD style='width:30px;'><H3>Decimals</H3></TD>
```

In the CSS we already have some code for a table header (TH), but it is clear you don't want that black background.
So maybe it is better to have some other style then, like a column header.
Then you can use something like this:
```html
<TD class='colheader'>Decimals</TD>
```
And then in the CSS add a `colheader` class where you make the font size slightly bigger and make it bold.

Reply:
"but it is clear you don't want that black background."
No... i guess you understood  me wrong. Look at the pictures. this change was made to put the "value" as a header in the next table where it belongs. nothing more. but in the process i hade to change the original header wich consisted "#",  "name", "formula" & "decimals". For consistency the had to look like a subheader....

the other change is just to make a colspan of the header of "normal" tables 100%... I thought the pictures made that clear.. it has nothing to do with css..

Reply:
I did understand you, I only tried to point out that we might need some other CSS style class between table header and table cell.
So this IMHO should be done by setting a class to that cell and then style it in the CSS.
This way one can later style it to whatever style is desired, as it has a class to it. If you explicitly do the markup like having the H3 markup added, then it might be more difficult to style it later.
For example it might look fine now, but if you plan on adding a dark theme, it might need some extra styling in the CSS as it is some style we don't have yet in the user interface.

N.B. I do agree it looks better on your screenshots, but I don't think it should be done like this in the code.